### PR TITLE
clean up the unnecessary flag for infra test

### PR DESCRIPTION
**Pull request by minchaow**
**How to categorize this PR?**

/area testing
/kind cleanup
/platform alicloud

**What this PR does / why we need it**:
This PR is to cleanup unnecessary flag for infrastructure integration test.
**Which issue(s) this PR fixes**:
Fixes #
None
**Special notes for your reviewer**:
None
**Release note**:

```other operator
None
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -154,4 +154,3 @@ integration-test-infra:
 		--access-key-id='$(shell cat $(ACCESS_KEY_ID_FILE))' \
 		--access-key-secret='$(shell cat $(ACCESS_KEY_SECRET_FILE))' \
 		--region=$(REGION) \
```
Comment (ialidzhikov) [inline Makefile:156]:
```suggestion
		--region=$(REGION)
```

Comment (minchaow) [reply Makefile:156]: Good catch! Thanks a lot.

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Review by jia-jerry — APPROVED** [review]

**Comment by ialidzhikov** [comment]
@minchaow , you need to rebase to latest master to fix the failing verify step.

/needs rebase

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**